### ISSUE TITLE
PDF-templates: Allow selection of page format and orientation per template

### ISSUE THREAD

**Issue by EZS-JD** [salesagility/SuiteCRM#628]
It would be a nice addition to the PDF-Templates module, if you could select the page format and orientation for each individual pdf-template (instead of changing it system-wide in the generatePdf.php script).
Relatedly, being able to choose a default page-size and orientation would be nice too, either via config.php or administration module.

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [PDF-templates: Allow selection of page format and orientation per template](https://trello.com/c/3NBkOsZ2/139-pdf-templates-allow-selection-of-page-format-and-orientation-per-template)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
